slamon_afm/models.py

```diff
@@ -7,10 +8,24 @@
     TypeDecorator
 from sqlalchemy.orm import relationship, backref
 from sqlalchemy.orm.exc import NoResultFound
-from sqlalchemy import func
+from sqlalchemy import func, event, case, literal_column
 
 db = SQLAlchemy()
 
+from slamon_afm.stats import statsd
+
+
+class TaskException(Exception):
+    task = None
+
+    def __init__(self, *args, task=None):
```

Comment:
To silence the flake8/pep8/pylint error "invalid syntax" (Which this most likely is), you need to change this line to:

```python
def __init__(self, task=None, *args):
```